ts/Core/Renderer/HTML/HTMLRenderer.ts

```diff
@@ -314,7 +314,7 @@ class HTMLRenderer extends SVGRenderer {
                             // updating the shadow div counterpart with the same
                             // style.
                             css: function (styles: CSSObject): HTMLElement {
-                                wrapper.css.call(parentGroup, styles);
+                                wrapper.css(styles);
```

Comment:
This is risky - currently it applies the change to the group's div counterpart, but now it is changed to apply to the actual wrapper. I can't find a fail case for it now, and the visual tests won't pick it up because they don't include the HTML content. But I don't think we can do it this way, we must assume there's a good reason why it currently applies the style to `parentGroup`.

Reply:
I see now that that code was new with https://github.com/highcharts/highcharts/commit/809fbbf37e321a63c73ae86e136774d056499400 . Did you bisect this regression to that commit? I'll have to do some more testing before approving.

Reply:
@bm64 bisected it [here](https://github.com/highcharts/highcharts/issues/19200#issuecomment-1608515450). This bug appeared exactly when adding the `css` method in this commit. It seemed to substitute the `css` method call for the wrapper and it was probably missing, so I called it that way.

Reply:
Thanks, I found out what's wrong. It doesn't take into account that there's a difference between SVGLabel.css and SVGElement.css. I'll upload a fix soon.